.github/ISSUE_TEMPLATE.md

```diff
@@ -1,5 +1,7 @@
 ## Environment
 
+Note you can also find versions at the /dashboard route of your Livebook.
```

Comment:
Maybe use a comment so it doesn't accidentally end up in the actual description?

```suggestion
<!-- Note: you can also find versions at the /dashboard route of your Livebook. -->
```